app/views/provider/admin/backend_apis/metrics/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<% if @metric.child? %>
```

Comment:
Same as in https://github.com/3scale/porta/pull/1103/files#diff-f8f3bc8ebf3c49aff1e52f17d2f2be6e